module/rdpCapture.c

```diff
@@ -586,6 +590,23 @@ isShmStatusActive(enum shared_memory_status status) {
     }
 }
 
+/******************************************************************************/
+/* copy rects with no error checking */
+static uint64_t
+wyhash_rfx_tile(const uint8_t *src, int src_stride, int x, int y, uint64_t seed)
+{
+    int row;
+    uint64_t hash;
+    const uint8_t *s8;
+    hash = seed;
+    for(row = 0; row < 64; row++)
+    {
+        s8 = src + (y+row) * src_stride + x * 4;
+        hash = wyhash((const void*)s8, 64 * 4, hash, _wyp);
+    }
+    return hash;
```

Comment:
Small point (and I haven't looked at the generated code), but you could make the loop tighter with:-

```c
    s8 = src + (y * src_stride) + (x * 4);
    for(row = 0; row < 64; row++)
    {
        hash = wyhash((const void*)s8, 64 * 4, hash, _wyp);
        s8 += src_stride;
    }
```

Reply:
Indeed, it might be optimized by the compiler though.